### ISSUE TITLE
Documentation's example is 100% wrong

### ISSUE THREAD

**Issue by mercmobily** [PolymerElements/iron-meta#36]
[Documentation in GitHub](https://github.com/PolymerElements/iron-meta) states:

> Pure imperative form would be like:
> `document.createElement('iron-meta').byKey('info');`

[Documentation in the elements' catalog](https://elements.polymer-project.org/elements/iron-meta) states:

> Pure imperative form would be like:
> `document.createElement('iron-meta').byKey('info').getAttribute('value');`

Which is totally wrong. The catalog's documentation is the first one I looked at, and I was going insane trying to understand why `byKey()` would ever return an *element*.

**Comment by mercmobily** [comment]
I assume the problem might stem from the sentence:

" I could define more attributes or use child nodes to define additional metadata."

Which I don't even understand...

**Comment by davidmaxwaterman** [comment]
Yes, I am wondering what 'use child nodes' means here...some example of this would be nice in the docs/demos.
